#5 doc: add prerequisites (#6)
derhuerst/berlin-gtfs-rt-server: readme.md

`berlin-gtfs-rt-server` needs access to a [Redis](https://redis.io/) server, you can configure a custom host/port by setting the `REDIS_URL` environment variable.

> It also needs access to a [PostgreSQL](https://www.postgresql.org) 14+ server with the [PostGIS](https://postgis.net/) extension activated; Pass custom [`PG*` environment variables](https://www.postgresql.org/docs/14/libpq-envars.html) if you run PostgreSQL in an unusual configuration.

Reviewer: I'm trying to make the distinction between installing an extension into the RDBMS and activating/loading it into a DB more clear. Unfortunately, not even the [`CREATE EXTENSION` docs](https://www.postgresql.org/docs/14/sql-createextension.html) are consistent about this.

```suggestion
It also needs access to a [PostgreSQL](https://www.postgresql.org) 14+ server with [PostGIS](https://postgis.net/) installed; Pass custom [`PG*` environment variables](https://www.postgresql.org/docs/14/libpq-envars.html) if you run PostgreSQL in an unusual configuration.
```

It also needs access to a [NATS Streaming](https://nats-io.gitbook.io/legacy-nats-docs/nats-streaming-server-aka-stan) server (just follow its [installation guide](https://nats-io.gitbook.io/legacy-nats-docs/nats-streaming-server-aka-stan/stan-nats-streaming-server/installing)); Set the `NATS_STREAMING_URL` environment variable if you run it in an unusual configuration.

> It also needs the command line instruction [Sponge](https://linux.die.net/man/1/sponge) from the [Moreutils](https://repology.org/project/moreutils/information) package installed on the host computer.

Reviewer: ```suggestion
It also needs the [`sponge` command](https://linux.die.net/man/1/sponge) from the [moreutils package](https://repology.org/project/moreutils/information) to be installed.
```
